docs/TabBarExample.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) "../src/less/index.less";
+
+.TabBarExample {
+  .margin--top--2xl;
+  font-size: 0.875rem;
```

Comment:
I tend to fall on the make the text bigger camp for legibility purposes even though I tend to prefer very small text sizes for myself.

10 / 12 / 14 / 16 feels like too many close type sizes to me and doesn't fit a scale very well. Will think on it.